lib/jira/commands/tickets.rb

```diff
@@ -0,0 +1,22 @@
+module Jira
+  class CLI < Thor
+
+    desc "tickets", "List the tickets of the input username"
```

Comment:
We need to handle pagination here. Looking at the API response for my user, JIRA auto-limits the response to the last 50 tickets.
```
{"expand":"schema,names","startAt":0,"maxResults":50,"total":2604, ...
```
What are your thoughts on this?